src/app/containers/manuscript/affiliations-list/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { IconButton } from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+
+import { SectionContainer } from 'app/components/section-container';
+import { ActionButton } from 'app/components/action-button';
+import { getAffiliations } from 'app/selectors/manuscript.selectors';
+import { Affiliation } from 'app/models/affiliation';
+import { useAffiliationStyles } from './styles';
+
+export const AffiliationsList: React.FC<{}> = () => {
+  const classes = useAffiliationStyles();
+  const affiliations = useSelector(getAffiliations);
+
+  const renderAffiliation = (aff: Affiliation) => (
+    <div key={aff.id} className={classes.listItem}>
+      <div className={classes.affiliationInfo}>
+        <sup className={classes.orderLabel}>({aff.label})</sup>
+        {[aff.institution.department, aff.institution.name, aff.address.city, aff.country].join(', ')}
```

Comment:
Would a template literal be better, or rather faster, here?

Reply:
actually. There's one part missing and that is filter the empty strings. The intention was to skip the missing fields and avoid commas.

Reply:
Ah, yeah even I didn't consider that. A filter or reduce should do it, then the join makes sense :-)